app/views/recipe_foods/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+  <body>
+    <div class='container'>
+      <h2>Update Ingredient:</h2>
+      <%= form_with model: @recipefoods, url: ingredient_update_path(@recipe.id, @recipefood.id), method: :put do |f| %>
+        <p><%= @recipefood.food.name %></p>
+        <%= f.label "Quantity:" %>
+        <%= f.number_field :quantity %>
+        <%= f.submit "Update Ingredient" %>
+      <% end %>
```

Comment:
- [ ] @asdt560 and @d4nQw3rty good work on your views however its best practice to use partial views for your form, kindly add it as you did for the foods.